specs/WinRT/MakeMSIX-spec.md

```diff
@@ -0,0 +1,1002 @@
+    }
+
+    /// Static methods for creating and unpacking packages.
+    [contract(MakeMSIXContract, 1)]
+    runtimeclass MakeMSIXManager
+    {
+        /// Creates packages from the inputPath as specified by the createPackageOptions.
+        /// Output path for the full packaged file.
+        static Windows.Foundation.IAsyncAction CreatePackage(
+            String inputPath,
+            String outputFileName,
+            CreatePackageOptions createPackageOptions);
+
+        /// Creates bundles from the inputPath as specified by the createBundleOptions.
+        /// Output path for the full bundled file.
+        static Windows.Foundation.IAsyncAction CreateBundle(
+            String inputPath,
+            String outputFileName,
+            CreateBundleOptions createBundleOptions);
+
+        /// Unpacks a package at inputFileName as specified by the extractPackageOptions.
+        /// The output folder to unpack the package into.
+        static Windows.Foundation.IAsyncAction ExtractPackage(
+            String inputFileName, 
+            String outputPath, 
+            ExtractPackageOptions extractPackageOptions);
+
+        /// Unbundles a package at inputFileName as specified by the extractBundleOptions.
+        /// The output folder to unpack the bundle package into.
+        static Windows.Foundation.IAsyncAction ExtractBundle(
+            String inputFileName,
+            String outputPath,
+            ExtractBundleOptions extractBundleOptions);
+
+        /// Creates a Kozani package from an existing package.
+        /// inputFileName can be an appx, appxbundle, msix, or msixbundle
+        /// Output path for the kozani packaged file.
+        /// Format must match input file. If inputFileName is an msix, then outputFileName must be an msix.
+        /// If inputFileName is an msixbundle, then outputFileName must be an msixbundle.
+        static Windows.Foundation.IAsyncAction CreateKozaniPackage(
+            String inputFileName,
+            String outputFileName,
+            CreateKozaniPackageOptions createKozaniPackageOptions);
+
+        /// Creates a mountable image that contains the packages at inputFileNames.
+        /// Valid file extensions for outputFileName are vhd, vhdx, and CIM.
+        /// Can be used for Azure App Attach.
+        static Windows.Foundation.IAsyncAction CreateMountableImage(
+            Windows.Foundation.Collections.IVector<String> inputFileNames,
+            String outputFileName,
+            CreateMountableImageOptions createMountableImageOptions);
+
+        /// Gets information about the contents of a package 
+        static Windows.Foundation.IAsyncOperation<PackageInformation> GetPackageInformation(String packageFileName);
+    };
+}
+```
+
+# 4. Appendix
+
+# 5. Existing interfaces and options
+
+The existing tools are:
+-   [Makeappx.exe](https://learn.microsoft.com/en-us/windows/win32/appxpkg/make-appx-package--makeappx-exe-)
+    Shipped in Windows' Platform SDK.
```

Comment:
SUGGESTION: Suggestion make these sub-bullets

See Preview -- indenting here just means "one paragraph, word wrap it for me"

e.g.

```
The existing tools are:
- [MakeAppx.exe] blahblah
  - Foo blah
  - Bar blah
  - Ack ack ack